source/SpriteParameters.cpp

```diff
@@ -0,0 +1,151 @@
+/* SpriteParameters.cpp
+Copyright (c) 2022 by XessWaffle
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "SpriteParameters.h"
+
+#include "Sprite.h"
+
+SpriteParameters::SpriteParameters()
+{
+}
+
+
+
+SpriteParameters::SpriteParameters(const Sprite *sprite)
+{
+	AnimationParameters initDefault;
+	static ConditionSet empty;
+	auto tuple = SpriteParameters::SpriteDetails{sprite, initDefault, empty};
```

Comment:
```suggestion
	auto tuple = SpriteParameters::SpriteDetails{sprite, initDefault, {}};
```
wouldnt this work too?